aws/aws.go

```diff
@@ -154,6 +154,19 @@ var APNortheast = Region{
 	"https://iam.amazonaws.com",
 }
 
+var APNortheast2 = Region{
+	"ap-northeast-2", // Asia Pacific (Seoul)
+	"https://ec2.ap-northeast-2.amazonaws.com",
+	"https://s3-ap-northeast-2.amazonaws.com",
+	"",
+	true,
+	true,
+	"",
+	"https://sns.ap-northeast-2.amazonaws.com",
+	"https://sqs.ap-northeast-2.amazonaws.com",
+	"https://iam.amazonaws.com",
+}
+
```

Comment:
I would suggest using the field names here just to make it clearer what these values mean. You can also then omit the default values (empty strings).